commit 6631363a5992a81d7bbcecaba68886564ad701ba
Author: Thomas Munro <tmunro@postgresql.org>
Date:   Sat Nov 25 10:39:32 2023 +1300

    Fix whitespace.
    
    Use tabs instead of spaces for new paragraph, like existing paragraphs.

diff --git a/src/port/pqsignal.c b/src/port/pqsignal.c
index 4ea2dea77e0..23e7f685c14 100644
--- a/src/port/pqsignal.c
+++ b/src/port/pqsignal.c
@@ -28,19 +28,19 @@
  *
  *	Windows, of course, is resolutely in a class by itself.  In the backend,
  *	this relies on pqsigaction() in src/backend/port/win32/signal.c, which
  *	provides limited emulation of reliable signals.
  *
- * 	Frontend programs can use this version of pqsignal() to forward to the
- * 	native Windows signal() call if they wish, but beware that Windows signals
- * 	behave quite differently.  Only the 6 signals required by C are supported.
- * 	SIGINT handlers run in another thread instead of interrupting an existing
- * 	thread, and the others don't interrupt system calls either, so SA_RESTART
- * 	is moot.  All except SIGFPE have SA_RESETHAND semantics, meaning the
- * 	handler is reset to SIG_DFL each time it runs.  The set of things you are
- * 	allowed to do in a handler is also much more restricted than on Unix,
- * 	according to the documentation.
+ *	Frontend programs can use this version of pqsignal() to forward to the
+ *	native Windows signal() call if they wish, but beware that Windows signals
+ *	behave quite differently.  Only the 6 signals required by C are supported.
+ *	SIGINT handlers run in another thread instead of interrupting an existing
+ *	thread, and the others don't interrupt system calls either, so SA_RESTART
+ *	is moot.  All except SIGFPE have SA_RESETHAND semantics, meaning the
+ *	handler is reset to SIG_DFL each time it runs.  The set of things you are
+ *	allowed to do in a handler is also much more restricted than on Unix,
+ *	according to the documentation.
  *
  * ------------------------------------------------------------------------
  */
 
 #include "c.h"
